setup.py

```diff
@@ -33,7 +33,7 @@
     ],
     entry_points={
         'console_scripts': [
-            'run-ci=run_ocsci:main',
+            'run-ci=ocsci.main:main',
```

Comment:
so run-ci should be broken in current master? @zmc the test you added is supposed to catch that right?

Reply:
Ahh ignore, you moved it out.